src/pepper/preprocessor.py

```diff
@@ -33,7 +34,8 @@ def main(args=None):
         args.output_file.write(output)
         args.output_file.close()
     else:
-        with open(os.path.split(args.input_file.name)[1] + ".preprocessed.cc", 'w') as output_file: # NOQA
+        basepath = os.path.split(args.input_file.name)[0]
+        with open(basepath + '/' + os.path.split(args.input_file.name)[1] + ".preprocessed.cc", 'w') as output_file: # NOQA
```

Comment:
There has got to be a os.path function to handle this mess

 - [ ] Fix this call to use some sane library call